pkgs/applications/office/appflowy/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchzip, autoPatchelfHook, makeWrapper, gtk3, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "appflowy";
+  version = "0.0.2";
+
+  src = fetchzip {
+    url = "https://github.com/AppFlowy-IO/appflowy/releases/download/${version}/AppFlowy-linux-x86.tar.gz";
+    sha256 = "1fvv4mlgf0vqcq5zh0zl2xr44saz0sm47r8whcywwrmcm0l66iv6";
+  };
+
+  nativeBuildInputs = [
+      autoPatchelfHook
+      makeWrapper
+  ];
+
+  buildInputs = [
+      gtk3
+      openssl
+  ];
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  installPhase = ''
+    mkdir -p $out/opt/
+    mkdir -p $out/share/applications/
+    mkdir -p $out/bin/
+
+    # Copy archive contents to the outpout directory
+    cp -r ./* $out/opt/
+
+    # Create the desktop file
+    echo -e "[Desktop Entry] \nName=AppFlowy \nComment=An Open Source Alternative to Notion \nIcon=$out/opt/flowy_logo.svg \nExec=$out/bin/appflowy \nCategories=Office \nType=Application \nTerminal=false" > $out/share/applications/appflowy.desktop
```

Comment:
Use the desktopItems attribute and `copyDesktopItems`. See [here](https://github.com/NixOS/nixpkgs/blob/75a86ab9f5691930f79de6c72e9f80eb280cf2dd/pkgs/games/dhewm3/default.nix#L27) and [here](https://github.com/NixOS/nixpkgs/blob/75a86ab9f5691930f79de6c72e9f80eb280cf2dd/pkgs/games/dhewm3/default.nix#L24) for an example

Reply:
I applied your change, but the `.desktop` file doesn't seem to work when I try to launch it through my file explorer. 
Is it normal ?

Reply:
It seems to start it for me, be sure to really install everything.

Reply:
Okay thanks :sweat_smile: